packages/camera/src/hooks/useAvailable/useAvailable.test.js

```diff
@@ -0,0 +1,10 @@
+import { renderHook } from '@testing-library/react-hooks';
+import UseAvailable from './index';
+
+describe('UseAvailable', () => {
+  jest.mock('../../__mocks__/expo-camera');
```

Comment:
Plz remove this mocking call from here and add it to root level. It should be called only once to mock every 3-party package in one go...